Formula/migrate.rb

```diff
@@ -0,0 +1,27 @@
+class Migrate < Formula
+  desc "Database migrations CLI tool"
+  homepage "https://github.com/golang-migrate/migrate"
+  url "https://github.com/golang-migrate/migrate.git",
+      :tag => "v3.3.0",
+      :revision => "bcd996f3df28363f43e2d0935484c4559537a3eb"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/golang-migrate/migrate").install buildpath.children
+
+    # Build and install cli as "migrate"
+    cd "src/github.com/golang-migrate/migrate" do
+      system "dep", "ensure", "-vendor-only"
+      system "make", "build-cli"
```

Comment:
This also runs the build for windows and linux targets.

Reply:
makefile provides v3.3.0(read from git tags), but does not provide single target build, i think it's acceptable trade some build time for simplicity

Reply:
We disagree. Please make sure it's only building for macOS.